### PR TITLE
remove customResource.json

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- remove `customResources.json` and the rest functionality of that,
- change JSON to YAML `errors.md`
- ...

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Extensibility/errors.md [hunk 1]
```diff
@@ -15,20 +15,16 @@ if (!structure.path && !structure.children && !Array.isArray(structure)) {
...
-}
+```yaml
+type: boolean
+name: isSecretRecipe
 ```
 
 he will see a notification like:
 
-```
+```yaml
```
Comment (Wawrzyn321) [inline core-ui/src/components/Extensibility/errors.md:25]: It's not a code snippet, just a excerpt of an error message. 
```suggestion
```
```

File: core/src/luigi-config/customResources.js [hunk 2]
```diff
@@ -137,12 +122,10 @@ export async function getCustomResources(authData) {
       ...busolaOwnExtConfigs,
       ...customResources[clusterName].flat(),
     ];
-
-    const clusterCustomResources = await loadBusolaClusterCRs();
+    
     const targetClusterCustomResources = await loadTargetClusterCRs(authData);
 
     const additionalExtResources = Object.values({
```
Comment (Wawrzyn321) [inline core/src/luigi-config/customResources.js:128]: Why the spread?

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**